services/guard/guard_test.go

```diff
@@ -0,0 +1,33 @@
+package guard
+
+import (
+	"testing"
+
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/network"
+	"github.com/dedis/cothority/sda"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestServiceGuard(t *testing.T) {
+	defer log.AfterTest(t)
+	log.TestOutput(testing.Verbose(), 4)
+	local := sda.NewLocalTest()
+	// generate 5 hosts, they don't connect, they process messages, and they
+	// don't register the tree or entitylist
+	_, el, _ := local.GenTree(5, false, true, false)
+	defer local.CloseAll()
+
+	// Send a request to the service
+	client := NewClient()
+	log.Lvl1("Sending request to service...")
+	UID := []byte("USER")
+	Epoch := []byte("EPOCH")
+	msg := network.Suite.Point()
+
```

Comment:
Add a comment (sth along the lines of) "Send message twice to see key didn't change in the same epoch" (or similar)